examples/notebooks/datastores/7_cmems_data_store.ipynb

```diff
@@ -0,0 +1,2538 @@
+       "med-cmcc-tem-an-fc-hts",
+       "med-cmcc-tem-an-fc-m",
+       "med-cmcc-cur-int-m",
+       "med-cmcc-cur-rean-d",
+       "med-cmcc-cur-rean-h",
+       "med-cmcc-cur-rean-m",
+       "med-cmcc-mld-int-m",
+       "med-cmcc-mld-rean-d",
+       "med-cmcc-mld-rean-m",
+       "med-cmcc-sal-int-m",
+       "med-cmcc-sal-rean-d",
+       "med-cmcc-sal-rean-m",
+       "med-cmcc-ssh-int-m",
+       "med-cmcc-ssh-rean-d",
+       "med-cmcc-ssh-rean-h",
+       "med-cmcc-ssh-rean-m",
+       "med-cmcc-tem-int-m",
+       "med-cmcc-tem-rean-d",
+       "med-cmcc-tem-rean-m",
+       "med-hcmr-wav-an-fc-h",
+       "med-hcmr-wav-rean-h",
+       "cmems_mod_med_wav_myint_4.2km_PT1H-i",
+       "dataset-armor-3d-nrt-monthly",
+       "dataset-armor-3d-nrt-weekly",
+       "dataset-armor-3d-rep-monthly",
+       "dataset-armor-3d-rep-weekly",
+       "dataset-sss-ssd-nrt-monthly",
+       "dataset-sss-ssd-nrt-weekly",
+       "dataset-sss-ssd-rep-monthly",
+       "dataset-sss-ssd-rep-weekly",
+       "cmems_obs-oc_atl_bgc-plankton_nrt_l3-multi-1km_P1D",
+       "cmems_obs-oc_atl_bgc-plankton_nrt_l3-olci-300m_P1D",
+       "cmems_obs-oc_atl_bgc-reflectance_nrt_l3-multi-1km_P1D",
+       "cmems_obs-oc_atl_bgc-reflectance_nrt_l3-olci-300m_P1D",
+       "cmems_obs-oc_atl_bgc-transp_nrt_l3-multi-1km_P1D",
+       "cmems_obs-oc_atl_bgc-transp_nrt_l3-olci-300m_P1D",
+       "cmems_obs-oc_atl_bgc-optics_nrt_l3-multi-1km_P1D",
+       "cmems_obs-oc_atl_bgc-plankton_my_l3-multi-1km_P1M",
+       "cmems_obs-oc_atl_bgc-reflectance_my_l3-multi-1km_P1M",
+       "cmems_obs-oc_atl_bgc-transp_my_l3-multi-1km_P1M",
+       "cmems_obs-oc_atl_bgc-optics_my_l3-multi-1km_P1M",
+       "cmems_obs-oc_atl_bgc-plankton_nrt_l4-multi-1km_P1M",
+       "cmems_obs-oc_atl_bgc-plankton_nrt_l4-olci-300m_P1M",
+       "cmems_obs-oc_atl_bgc-transp_nrt_l4-olci-300m_P1M",
+       "cmems_obs-oc_atl_bgc-plankton_my_l4-multi-1km_P1M",
+       "cmems_obs-oc_atl_bgc-plankton_my_l4-multi-climatology-1km_P1D",
+       "DMI-BALTIC-SST-L3S-NRT-OBS_FULL_TIME_SERIE",
+       "cmems_obs_oc_nws_bgc_tur-spm-chl_nrt_l3-hr-mosaic_P1D-m",
+       "cmems_obs_oc_nws_bgc_tur-spm-chl_nrt_l4-hr-mosaic_P1D-m",
+       "DMI-ARC-SEAICE_BERG_IW-L4-NRT-OBS",
+       "DMI-ARC-SEAICE_BERG_MOSAIC_IW-L4-NRT-OBS",
+       "DMI-ARC-SEAICE_BERG_MOSAIC-L4-NRT-OBS",
+       "DMI-ARC-SEAICE_BERG-L4-NRT-OBS"
+      ],
+      "text/plain": [
+       "<IPython.core.display.JSON object>"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {
+      "application/json": {
+       "expanded": false,
+       "root": "root"
+      }
+     },
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "JSON(list(store.get_data_ids()))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4df9dd30-c230-4bdb-8d5d-725deeb36753",
+   "metadata": {},
+   "source": [
+    "#### Get more info about a specific dataset. This includes a description of the possible open formats:"
```

Comment:
```suggestion
    "#### Get more info about a specific dataset."
```
I realize you just copied this from other notebooks, but I don't know what is meant with "open formats" here.